src/lib/draw-and-pick.js

```diff
@@ -19,7 +19,7 @@
 // THE SOFTWARE.
 
 /* global window */
-import {GL, withParameters, setParameters} from 'luma.gl';
+import {GL, withParameters, setParameters, NULL_PICKING_COLOR} from 'luma.gl';
```

Comment:
Not sure about this, i.e. that we should enforce a specific `NULL_PICKING_COLOR`, or even export it from luma.gl. 

The original system allowed the user of the picking module to specify it.